src/components/navigation-accordion/navigation-accordion.js

```diff
@@ -0,0 +1,106 @@
+  render() {
+    const { props } = this;
+    const secondLevelContent =
+      props.contents.seccondLevelItems &&
+      props.contents.seccondLevelItems.map(item => {
+        return (
+          <li key={item.slug} className="mt6">
+            <a
+              href={`#${item.slug}`}
+              className="color-blue-on-hover text-decoration-none unprose"
+            >
+              {item.text}
+            </a>
+          </li>
+        );
+      });
+    const firstLevelContent = props.contents.firstLevelItems.map(
+      (page, index) => {
+        const title = page.title;
+        let icon = null;
+        const isActive = this.props.currentPath === page.path;
+        const breakLineClasses = classnames('py3 ', {
+          'border-t border--gray-light': index !== 0
+        });
+        const textClasses = classnames('pl12 py12 txt-bold txt-m flex-child', {
+          'color-black pb3': isActive
+        });
+        const activeSectionClasses = classnames(
+          'pl24-mm px0 block-mm none pr12',
+          {
+            'bg-lighten75': isActive
+          }
+        );
+        if (!isActive) {
+          icon = (
+            <Icon
+              name="chevron-down"
+              className="flex-child flex-child--no-shrink icon color-gray h24 w24"
+            />
+          );
+        }
+        return (
+          <div key={index} className={activeSectionClasses}>
+            <div className={breakLineClasses}>
+              <a
+                href={page.path}
+                className="color-blue-on-hover color-gray text-decoration-none unprose flex-parent flex-parent--space-between-main flex-parent--center-cross"
+              >
+                <div className={textClasses}>{title}</div>
+                {icon}
+              </a>
+              {isActive && secondLevelContent ? (
+                <div className="ml24 pt0">
+                  <ul className="txt-m pb12 inline-block-mm none color-blue-on-hover-anchor unprose">
+                    {secondLevelContent}
+                  </ul>
+                </div>
+              ) : (
+                ''
+              )}
+            </div>
+          </div>
+        );
+      }
+    );
+    return (
+      <div>
+        <div className="block-mm none">{firstLevelContent}</div>
+        <div className="none-mm block bg-gray-faint">
+          <NavigationDropdown
+            currentPath={props.currentPath}
+            dropdownOptions={props.contents.firstLevelItems}
+          />
+        </div>
+      </div>
+    );
+  }
+}
+
+NavigationAccordion.propTypes = {
+  currentPath: PropTypes.string,
+  contents: PropTypes.shape({
+    firstLevelItems: PropTypes.arrayOf(
+      PropTypes.shape({
+        title: PropTypes.string.isRequired,
+        path: PropTypes.string.isRequired
+      })
+    ).isRequired,
+    seccondLevelItems: PropTypes.arrayOf(
+      PropTypes.shape({
+        level: PropTypes.number.isRequired,
+        slug: PropTypes.string.isRequired,
```

Comment:
Could we use `path` in the same way as for `firstLevelItems`? Or how about `url` or `href`? Not sure we *need* to build in the assumption that second-level items are always hash links.

Reply:
Sure. That's just the shape that's collected via Batfish. I suppose I could rename the names in the PageShell before passing here?

Reply:
Yeah, and when passing it in you'd also prefix the `#`.